documentation/markdown/features.md

```diff
@@ -0,0 +1,90 @@
+# Features
+
+Below is a non-exhaustive listing of the features available to a server instance,
+depending on the chosen configuration.
+The core feature of the CSS is how it uses Components.js to configure its components,
+so any of the features below can always be adapted or replaced with custom components if required.
+See this [tutorial](https://github.com/CommunitySolidServer/tutorials/blob/main/custom-configurations.md)
+and/or this [example repository](https://github.com/CommunitySolidServer/hello-world-component)
+for more information on that.
+
+To generate configurations with some of these features enabled/disabled,
+you can use the [configuration generator](https://communitysolidserver.github.io/configuration-generator/).
+
+## Authentication
+
+Clients are identified based on the contents of DPoP tokens,
+as described in the [Solid-OIDC specification](https://solidproject.org/TR/oidc).
+
+Alternatively, for debugging purposes the server can be configured to always identify the client as a fixed WebID,
```

Comment:
The way it is currently written, sounds very limiting. I would use a couple of more words to describe that the DI framework also allows configuring dummy components for debugging, development, or experimentation purposes. One of those options is a fixed WebID, others are…